swift/Arrow/Package.swift

```diff
@@ -32,7 +32,7 @@ let package = Package(
             targets: ["Arrow"]),
     ],
     dependencies: [
-        .package(url: "https://github.com/google/flatbuffers.git", from: "23.3.3")
+        .package(url: "https://github.com/google/flatbuffers.git", branch: "master")
```

Comment:
Could you add a comment why we need to use `master` for now and when we can specify a tag again?